src/enhanced-textarea.jsx

```diff
@@ -2,6 +2,28 @@ const React = require('react');
 const StylePropable = require('./mixins/style-propable');
 const AutoPrefix = require('./styles/auto-prefix');
 
+const rowsHeight = 24;
+
+const styles = {
```

Comment:
I see that you made the `styles` global.

Do you think in the future we could use some styles from the theme for this component? That would cause issues with this global approach right? Nearly all components implement a `getStyles()` method. I think we should be consistent about that. What do you think @oliviertassinari?